src/Roadkill.Web/App_Data/NLog.config

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<nlog xmlns="http://www.nlog-project.org/schemas/NLog.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+	<targets>
+		<target name="FileTarget" xsi:type="File"
+			layout="[${longdate:universalTime=True}] [${level}] ${message}"
```

Comment:
From the feedback not many people (except devs) use the logging feature, so I'm hoping they can read up on the NLog target/logger specs instead of Roadkill giving a dumbed-down version of them.